src/bin/cargo/cli.rs

```diff
@@ -244,5 +240,23 @@ See 'cargo help <command>' for more information on a specific command.\n",
                 .number_of_values(1)
                 .global(true),
         )
-        .subcommands(commands::builtin())
+        .subcommands(
+            commands::builtin()
+                .into_iter()
+                .map(|c| add_quiet(c))
+                .collect::<Vec<_>>(),
+        )
+}
+
+// add --quiet option to command
```

Comment:
Added this because `quiet` was already present (set to global) for all `cargo` subcommands https://github.com/rust-lang/cargo/pull/6358/files#diff-56e47269168a97da06bace967cc234e4L229. I'm adding this to all subcommands except those that already have this option defined (as currently couldn't find a way to override the description for a global option for `clap.rs`). One other option would be to explicitly add `--quiet` for all other subcommands.

Reply:
Hm ok, if we end up landing this I think we'll want to take the route of declaratively adding `--quiet` to each subcommand, either through a shared implementation or manually, I don't think we'll want to sneak in `--quiet` as an argument right at the end

Reply:
Thanks, makes sense. Also I noticed `--quiet` does nothing for some subcommands i.e `cargo search`, `generate-lockfile`, etc. Is it ok to not add the `--quiet` for these? The option was present for all before so my worry is removing it may cause breakage.

Reply:
Nah I think it's fine to add this for those commands to, they'll likely just pick up the existing behavior for `--quiet` naturally!